src/store/modules/substance.js

```diff
@@ -89,6 +90,7 @@ let actions = {
             color: "warning",
             dismissCountDown: 4
           };
+          // ?: should getting a failed search result clear what you have loaded?
```

Comment:
Should these 'clear events' happen when there is a failed search? it seems like this behavior should be changed. Not sure if this is the right time or if we need a ticket with 1 point